edgeNodeSetup.sh

```diff
@@ -93,6 +94,31 @@ fi
 export JAVA_HOME=/usr/lib/jvm/java-7-openjdk-amd64
 echo "JAVA_HOME=$JAVA_HOME">>/etc/environment
 
+echo "Getting list of hosts from ambari"
+hostsJson=$(curl -u $clusterLogin:$clusterPassword -X GET https://$clustername.azurehdinsight.net/api/v1/clusters/$clustername/hosts)
+echo $hostsJson
+
+echo "Parsing list of hosts"
+hosts=$(echo $hostsJson | sed 's/\\\\\//\//g' | sed 's/[{}]//g' | awk -v k="text" '{n=split($0,a,","); for (i=1; i<=n; i++) print a[i]}' | sed 's/\"\:\"/\|/g' | sed 's/[\,]/ /g' | sed 's/\"//g' | grep -w 'host_name')
+echo $hosts
+
+echo "Extracting headnode0 and headnode1"
+headnode0=$(echo $hosts | grep -Eo '\bhn0-([^[:space:]]*)\b')
+headnode1=$(echo $hosts | grep -Eo '\bhn1-([^[:space:]]*)\b')
+echo "headnode0: $headnode0, headnode1: $headnode1"
+
+echo "Extracting headnode0 and headnode1 IP addresses"
+headnode0ip=$(dig +short $headnode0)
```

Comment:
why use ip address? I don't think they are static, the host name should work fine.